src/lib/efl/meson.build

```
@@ -8,7 +8,7 @@ efl_header_src = [
   'Efl_MVVM_Common.h'
 ]
 
-efl_src = []
+efl_src = ['efl_gfx_error.c']
```

```suggestion
efl_src = files('efl_gfx_error.c')
```

This should get the reference to the file properly.